cmd/cmd_root.go

```diff
@@ -31,6 +31,27 @@ func GetRootCommand() *cobra.Command {
 				}
 			}
 
+			showErrors, err := cmd.Flags().GetBool("show-errors")
```

Comment:
isn't this being handled by `globalFlags.MergeToConfig(config)`?

Reply:
@fabioDMFerreira oh you right ... I will move that line outside the `if` block

Reply:
Fixed by https://github.com/nanovms/ops/pull/1033/commits/70a813bb5130c1f9011f2d055f004716c5e21bf3